run_nextflow_dims.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -euo pipefail
+
+workflow_path='/hpc/dbg_mz/development/DIMS_Nextflow'
+
+# Set input and output dirs
+input=$1
+output=$2
+email=$3
+samplesheet=$4
+nr_replicates=$5
+resolution=$6
+ppm=$7
+zscore=$8
+echo "input directory: $input"
+echo "output directory: $output"
+echo "workflow path: $workflow_path"
```

Comment:
There are currently 8 parameters given when starting the Nextflow pipeline; this will change when the pipeline is started using the GUI. Fixed parameters should be in the config file; parameters that vary for each dataset should be given via the GUI. A user should still be able to start the pipeline from the command line, though.
The echo lines can be omitted at a later stage.